pkg/fission-cli/logdb/influxdb.go

```diff
@@ -66,13 +66,18 @@ func (influx InfluxDB) GetLogs(filter LogFilter) ([]LogEntry, error) {
 	parameters["time"] = timestamp
 	//the parameters above are only for the where clause and do not work with LIMIT
 
+	orderCondition := ""
```

Comment:
We can set default query as `orderCondition := " order by \"time\" asc"`

Reply:
Since the problem mentioned in https://github.com/fission/fission/pull/405 got solved in influxdb 1.2 (https://github.com/influxdata/influxdb/pull/7907)
Would you mind to remove the code `sort.Slice` as well? because it will change the order of log output.
https://github.com/fission/fission/blob/58811ab522774ec04d9bdbf4440acfd718f0757d/pkg/fission-cli/logdb/influxdb.go#L128-L137